docker/config.deco.json

```diff
@@ -1,4 +1,54 @@
 { 
+  "accounts": {
+    "spinup": {
+      "runners": ["dummyRunner", "instanceRunner"]
+    },
+    "spinupsec": {
+      "runners": ["dummyRunner", "instanceRunner"]
+    }
+  },
+  "jobRunners": {
+    "dummyRunner": {
+      "type": "dummy",
+      "config": {
+        "template": "Hello, {{.Account}}!"
+      }
+    },
+    "instanceRunner": {
+      "type": "instance",
+      "config": {
+        "endpointTemplate": "{{ .instancerunner_endpoint_tmpl }}",
+        "token": "{{ .instancerunner_token }}"
+      }
+    }
+  },
+  "jobsRepository": {
+    "type": "s3",
+    "refreshInterval": "{{ .jobs_repository_refresh_interval }}",
+    "config": {
+      "region": "us-east-1",
+      "akid": "{{ .jobs_repository_akid }}",
+      "secret": "{{ .jobs_repository_secret }}",
+      "bucket": "{{ .jobs_repository_bucket }}",
+      "prefix": "jobs"
+    }
+  },
+  "queueProvider": {
+    "type": "redis",
+    "config": {
+      "host": "{{ .queue_host }}",
+      "port": "{{ .queue_port }}",
+      "database": "{{ .queue_db }}"
```

Comment:
Probably doesn't matter here, but can these be authenticated?

Reply:
yes in the code, not easily in the redis implementation we're using

Reply:
if network access to that redis is relatively restricted it should be fine